gradio/routes.py

```diff
@@ -273,32 +273,20 @@ async def proxy_to_node(
         new_request = App.client.build_request(
             request.method, httpx.URL(url), headers=headers
         )
-        node_response = await App.client.send(new_request)
+        node_response = await App.client.send(new_request, stream=True)
         content = node_response.content
         user_agent = request.headers.get("user-agent", "").lower()
         is_safari = (
             "safari" in user_agent
             and "chrome" not in user_agent
             and "chromium" not in user_agent
         )
-        response_headers = {}
```

Comment:
I've removed this because it seems to work without, Do we remember why it's was here in the first place ?

Reply:
This was needed because SSR wasn't working locally with Safari without these changes. Let me test out this PR to see if its still needed